src/main/java/org/jabref/gui/entryeditor/LatexCitationsTabViewModel.java

```diff
@@ -77,6 +85,16 @@ public void init(BibEntry entry) {
         currentEntry = entry;
         Optional<String> citeKey = entry.getCitationKey();
 
+        // make sure that we only add the listener once at the beginning
+        if (!hasListener) {
+            try {
+                fileUpdateMonitor.addListenerForFile(directory.get(), this::refreshLatexDirectory);
+                hasListener = true;
+            } catch (IOException e) {
+                throw new RuntimeException(e);
```

Comment:
Where does it come from that contributors crash the whole application, which leads to data loss?

`LOGGER.error(...)` is the better way.